root/static/scripts/common/entity.js

```diff
@@ -412,7 +412,9 @@ import formatTrackLength from './utility/formatTrackLength';
             this.tracks = _.map(data.tracks, x => new Track(x));
 
             const positionArgs = {
-                medium_format: this.format,
+                medium_format: this.format
+                    ? lp_attributes(this.format.name, 'medium_format')
+                    : null,
```

Comment:
Can't this use the stuff from 312a3a7b0e3290a9eea80d00ddb612f41df82798 ?

Reply:
See 6cbcb2ee3433264f533e1ec8d4d61a873613e2c4

This changes the strings used.

Reply:
This seems sensible and less prone to errors IMO, if @yvanzo is happy with the string change (should only change the way strings are built, but not the final result).

Reply:
I’m happy! :grinning: